src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CommonField.java

```diff
@@ -29,6 +30,10 @@
 
     void setDisplayContextParameter(String displayContextParameter);
```

Comment:
we have a model object DisplayContextParameter. Can we use that instead of String? it would reduce the boilerplate in the code